ui/src/app/pages/za-settings/modals/create-car-questionnaire/create-car-questionnaire.component.ts

```diff
@@ -254,15 +252,15 @@ export class CreateCarFormComponent implements OnInit {
 
   createTestData() {
     this.inspectionForm.patchValue({
-      ...CarFormEnumsStrings.Inspection,
+      ...CarQuestionnaireEnumsStrings.Inspection,
       date: (new Date()).toLocaleDateString('ru-RU'),
       vin: CryptoJS.AES.encrypt( new Date().toString(), 'vin').toString().slice(0,17),
     });
     this.generalConditionForm.patchValue({
-      ...CarFormEnumsStrings.GeneralCondition
+      ...CarQuestionnaireEnumsStrings.GeneralCondition
     });
     this.carQuestionnaireForm.patchValue({
-      ...CarFormEnumsStrings.CarQuestionnaire
+      ...CarQuestionnaireEnumsStrings.CarQuestionnaire
     });
   }
 
```

Comment:
Need to refactor this component, replace and rename some functions for with field enums